Editor/Resources/EditorWindow/common.uss

```diff
@@ -17,14 +17,17 @@
     --color-input-label: #c4c4c4;
     --color-text-input-background: #2a2a2a;
     --color-text-input-border: #0d0d0d;
+    --color-card-border: #0d0d0d;
+    --color-card-background: #2a2a2a;
 
-    --font-size-small: 12px;
-    
+    --font-size-hint: 12px;
```

Comment:
To avoid changing the font size of this hint text, I added another property during the merge conflict resolution